model_analyzer/config/input/config_command_profile.py

```diff
@@ -294,8 +294,77 @@ def _add_profile_models_configs(self):
                             type_=str))
                 for k in PerfAnalyzerConfig.allowed_keys()
             })
+
         triton_server_environment_scheme = ConfigObject(
             schema={'*': ConfigPrimitive(str)})
+
+        # This comes from /usr/local/lib/python3.8/dist-packages/docker/models/containers.py
```

Comment:
/usr/local/bin is a local path on your machine, right? Is there a better way to point to the source?

Reply:
Correct. It's an installed package. I've updated the message and removed reference to my local path